.github/actions/multi-approvers/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "alwaysStrict": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "commonjs",
+    "noImplicitAny": true,
+    "outDir": "./.tsc",
+    "rootDir": "./",
```

Comment:
We typically want a `src` and `dist` dir here.

Reply:
I can address this once we resolve [this comment](https://github.com/abcxyz/pkg/pull/403#discussion_r2032267507).

Reply:
Done.